ddtrace/contrib/responder/patch.py

```diff
@@ -0,0 +1,109 @@
+
+# stdlib
+
+# project
+import ddtrace
+from ddtrace.ext import http
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+from ddtrace.utils.wrappers import unwrap as _u
+from ddtrace.propagation.http import extract_context_from_http_headers
+
+# 3p
+import responder
+patches = []  # will be overriden when patched funcs are defined.
+
+
+def patch():
+    if _is_patched(responder):
+        return
+    _set_patched(responder, True)
+
+    for obj, mname, fname, func in patches:
+        _w(mname, fname, func)
+
+
+def unpatch():
+    if not _is_patched(responder):
+        return
+    _set_patched(responder, False)
+
+    for obj, _, prop, _ in patches:
+        if '.' in prop:
+            attr, _, prop = prop.partition('.')
+            obj = getattr(obj, attr, object())
+        _u(obj, prop)
+
+
+async def _api_call(wrapped, instance, args, kwargs):
+    tracer = _get_tracer(instance)
+    assert tracer
+
+    scope, receive, send = args  # FIXME: what if this is partially using kwargs?
```

Comment:
We should be good since `API.__call__` only uses positional arguments:

https://github.com/taoufik07/responder/blob/master/responder/api.py#L361

```
async def __call__(self, scope, receive, send):
```